src/lib/core/coio.cc

```diff
@@ -522,12 +522,13 @@ coio_service_init(struct coio_service *service, const char *name,
 	service->handler_param = handler_param;
 }
 
-void
+int
 coio_service_start(struct evio_service *service, const char *uri)
```

Comment:
This function is unused, maybe drop id?

Reply:
I tried, but @alyapunov opposed: https://github.com/tarantool/tarantool/pull/6583#discussion_r744651864

`coio_service` is unusable, you're right, but let's fix it when we actually need to use it.